{2} setUpClass (tempest.api.compute.volumes.test_attach_volume.AttachVolumeMultiAttachTest) ... SKIPPED: Neutron is not available
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.177497s] ... ok
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.014533s] ... ok
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.012502s] ... ok
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.140238s] ... ok
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.009333s] ... ok
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.010304s] ... ok
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.106593s] ... ok
{2} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.047826s] ... ok
{2} tempest.api.volume.admin.test_group_type_specs.GroupTypeSpecsTest.test_group_type_specs_create_show_update_list_delete [0.594142s] ... ok
{1} tempest.api.compute.volumes.test_volume_snapshots.VolumesSnapshotsTestJSON.test_volume_snapshot_create_get_list_delete [6.953718s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list [0.068313s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_limit [0.068782s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_offset_and_limit [0.137062s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_limit [0.072098s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_offset_and_limit [0.132676s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_details [0.069167s] ... ok
{2} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_create_list_update_show_delete [0.376989s] ... ok
{2} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_list_by_optional_params [0.250206s] ... ok
{1} tempest.api.compute.volumes.test_volumes_get.VolumesGetTestJSON.test_volume_create_get_delete [2.826766s] ... ok
{0} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_compare_volume_stats_values [1.062223s] ... ok
{0} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_get_capabilities_backend [0.536446s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_encrypted_volumes_extend.EncryptedVolumesExtendAttachedTest) ... SKIPPED: EncryptedVolumesExtendAttachedTest skipped as Neutron is not available
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_create_show_list_delete [8.381644s] ... ok
{1} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsV319Test.test_reset_group_snapshot_status [9.577358s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_update [10.486289s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_create_group_from_group_snapshot [12.813619s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_associate_disassociate_qos [0.664643s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_back_end_consumer [0.177353s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_both_consumer [0.176857s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_front_end_consumer [0.176786s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_get_qos [0.025767s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_list_qos [0.027765s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_set_unset_qos_key [0.162753s] ... ok
{1} tempest.api.volume.admin.test_groups.GroupsV314Test.test_create_group_from_group [11.567743s] ... ok
{2} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_show_default_quota [0.348548s] ... ok
{2} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_update_default_quota [1.514332s] ... ok
{1} tempest.api.volume.admin.test_groups.GroupsV320Test.test_reset_group_status [3.260149s] ... ok
{1} setUpClass (tempest.api.volume.admin.test_multi_backend.VolumeMultiBackendTest) ... SKIPPED: Cinder multi-backend feature disabled
{1} setUpClass (tempest.api.volume.admin.test_snapshot_manage.SnapshotManageAdminTest) ... SKIPPED: SnapshotManageAdminTest skipped as Neutron is not available
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_delete_group_snapshots_following_updated_volumes [17.398960s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_delete_quota [0.688060s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_default_quotas [0.108792s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_quotas [0.123054s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage [4.219587s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_reset_snapshot_status [0.239337s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_group_snapshot_create_show_list_delete [8.992007s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_creating [3.567826s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage_after_volume_transfer [5.377241s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_show_quota_usage [0.435278s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_update_all_quota_resources_for_tenant [0.412972s] ... ok
{2} setUpClass (tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithMigrationTest) ... SKIPPED: Cinder multi-backend feature disabled
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_deleting [3.570691s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_delete_message [2.142407s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_list_show_messages [1.916302s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error [3.627484s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error_deleting [3.616846s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_update_snapshot_status [0.253210s] ... ok
{0} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_list_hosts [0.208491s] ... ok
{0} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_show_host [0.090868s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_volume_manage.VolumeManageAdminTest) ... SKIPPED: Manage volume tests are disabled
{2} tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithoutMigrationTest.test_available_volume_retype [5.388767s] ... ok
{0} tempest.api.volume.admin.test_volume_pools.VolumePoolsAdminTestsJSON.test_get_pools_without_details [0.735155s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_invalid_body [0.016263s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_none_body [0.018989s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_nonexistent_type_id [0.040509s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_delete_nonexistent_volume_type_id [0.021303s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_extra_spec_name [0.028319s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_volume_type_id [0.024082s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_list_nonexistent_volume_type_id [0.022833s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_multiple_extra_spec [0.020869s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_no_body [0.019092s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_none_extra_spec_id [0.039720s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_extra_spec_id [0.042630s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_type_id [0.040467s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volume_gigabytes [0.403529s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volumes [0.388436s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_host_name [0.028571s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_and_host_name [0.035010s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_binary_name [0.024722s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_volume_extend_gigabytes_quota_deviation [0.291473s] ... ok
{2} tempest.api.volume.test_versions.VersionsTest.test_list_versions [0.013723s] ... ok
{2} tempest.api.volume.test_versions.VersionsTest.test_show_version [0.083616s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_volume_host_name [4.009837s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_list_services [0.027142s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_log_reason_with_no_reason [0.023344s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_service_with_invalid_binary [0.026597s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_enable_service_with_invalid_host [0.027250s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_freeze_host_with_invalid_host [0.024349s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_thaw_host_with_invalid_host [0.026394s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_create_get_delete [0.370921s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_encryption_create_get_update_delete [0.399000s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_list [0.026854s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_update [0.309425s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_delete_cascade [8.850714s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_spec_create_get_delete [0.164528s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_list [0.113277s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_update [0.227686s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{1} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_gigabytes_snapshots [0.540518s] ... ok
{1} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_snapshots [0.105819s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_from_snapshot_cascade_delete [9.737299s] ... ok
{1} tempest.api.volume.admin.test_volume_type_access.VolumeTypesAccessTest.test_volume_type_access_list [0.511382s] ... ok
{0} tempest.api.volume.admin.test_volumes_list.VolumesListAdminTestJSON.test_volume_list_param_tenant [3.166277s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_volume_type_encryption_nonexistent_type_id [0.162000s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_empty_name [0.017468s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_repeated_name [0.157361s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_delete_nonexistent_type_id [0.020956s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_get_nonexistent_type_id [0.020421s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_get_list_accept_volume_transfer [6.075131s] ... ok
{1} tempest.api.volume.test_availability_zone.AvailabilityZoneTestJSON.test_get_availability_zone_list [0.195062s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_list_delete_volume_transfer [4.022565s] ... ok
{2} setUpClass (tempest.api.volume.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{0} tempest.api.volume.test_extensions.ExtensionsTestJSON.test_list_extensions [0.193257s] ... ok
{1} setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata) [0.000000s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}

{2} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume [0.205075s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-20 17:37:49,058 65288 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.201s
2025-02-20 17:37:49,058 65288 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-6a4b53f5-6af4-4d46-a3cf-7243a640844e', 'date': 'Thu, 20 Feb 2025 17:37:49 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{0} tempest.api.volume.test_volume_absolute_limits.AbsoluteLimitsTests.test_get_volume_absolute_limits [0.131751s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_crud_snapshot_metadata [0.255688s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_update_show_snapshot_metadata_item [0.229290s] ... ok
{2} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_volume [8.533123s] ... ok
{0} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_crud_volume_metadata [0.354932s] ... ok
{0} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_update_show_volume_metadata_item [0.233330s] ... ok
{1} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend [0.869577s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-20 17:38:02,260 65286 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.867s
2025-02-20 17:38:02,260 65286 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-b237b08d-4f1b-464a-9b77-bbde9a1f9e22', 'date': 'Thu, 20 Feb 2025 17:38:02 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{2} tempest.api.volume.test_volumes_clone_negative.VolumesCloneNegativeTest.test_create_from_volume_decreasing_size [4.402130s] ... ok
{2} tempest.api.volume.test_volumes_get.VolumesSummaryTest.test_show_volume_summary [0.202463s] ... ok
{1} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend_when_volume_has_snapshot [8.512062s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_get_list_accept_volume_transfer [5.566943s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_deactivated_image [0.566396s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_image_with_decreasing_size [0.398010s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.016884s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_snapshot_id [0.027596s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_source_volid [0.033079s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_volume_type [0.033050s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_negative [0.016401s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.016013s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.017012s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.029092s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.011725s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_detach_volumes_with_invalid_volume_id [0.028729s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_invalid_volume_id [0.028220s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.012129s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_invalid_status [0.050705s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_nonexistent_name [0.061495s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_invalid_status [0.035267s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_nonexistent_name [0.036573s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_negative_volume_status [0.139463s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_nonexistent_volume_id [0.027815s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_unreserve_volume_with_nonexistent_volume_id [0.027311s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_empty_volume_id [0.011433s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_invalid_volume_id [0.034863s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_nonexistent_volume_id [0.028044s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.028080s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_None_size [0.016538s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_non_number_size [0.015448s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_nonexistent_volume_id [0.033442s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_size_smaller_than_original_size [0.014956s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_without_passing_volume_id [0.030321s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.048760s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_list_delete_volume_transfer [4.049176s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete [7.670004s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_get_list_accept_volume_transfer [4.533566s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit [0.031943s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_infinite [0.066785s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_zero [0.025396s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_marker [0.063443s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_offset [0.025618s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_asc [0.027802s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_desc [0.025607s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_asc [0.024778s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_desc [0.028667s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_asc [0.030966s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_desc [0.029468s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_details_with_params [0.084881s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_with_params [0.083453s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_list_delete_volume_transfer [3.951068s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_as_clone [12.281301s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image [0.020343s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-20 17:38:33,859 65286 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.018s
2025-02-20 17:38:33,859 65286 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-c916b018-5a53-4e44-9a32-68076208bb16', 'date': 'Thu, 20 Feb 2025 17:38:33 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{0} setUpClass (tempest.api.volume.test_volumes_extend.VolumesExtendAttachedTest) ... SKIPPED: VolumesExtendAttachedTest skipped as Neutron is not available
{0} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_with_nonexistent_volume_id [0.085807s] ... ok
{0} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_without_passing_volume_id [0.032401s] ... ok
{0} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshot_invalid_param_limit [0.019541s] ... ok
{0} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_marker [0.024664s] ... ok
{0} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_sort [0.016006s] ... ok
{0} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_volume_from_snapshot_decreasing_size [7.211569s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list [0.041349s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_by_name [0.038855s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_by_name [0.056874s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_pagination [0.267422s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_with_multiple_params [0.121839s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_pagination [0.194325s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_param_display_name_and_status [0.037544s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_display_name_and_status [0.059537s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_marker [0.073907s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_metadata [0.078309s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_details [0.062495s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_param_metadata [0.044227s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_availability_zone [0.082153s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_bootable [0.078823s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_status [0.080869s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_availability_zone [0.064342s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_bootable [0.063329s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_status [0.063483s] ... ok
{1} setUpClass (tempest.api.volume.test_volumes_snapshots.VolumesSnapshotTestJSON) ... SKIPPED: VolumesSnapshotTestJSON skipped as Neutron is not available

==============================
Failed 4 tests - output below:
==============================

setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata)
------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume[id-cbbcd7c6-5a6c-481a-97ac-ca55ab715d16,image]
--------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-20 17:37:49,058 65288 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.201s
2025-02-20 17:37:49,058 65288 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-6a4b53f5-6af4-4d46-a3cf-7243a640844e', 'date': 'Thu, 20 Feb 2025 17:37:49 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend[id-9a36df71-a257-43a5-9555-dc7c88e66e0e]
--------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-20 17:38:02,260 65286 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.867s
2025-02-20 17:38:02,260 65286 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-b237b08d-4f1b-464a-9b77-bbde9a1f9e22', 'date': 'Thu, 20 Feb 2025 17:38:02 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image[id-54a01030-c7fc-447c-86ee-c1182beae638,image,smoke]
-------------------------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-20 17:38:33,859 65286 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.018s
2025-02-20 17:38:33,859 65286 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-c916b018-5a53-4e44-9a32-68076208bb16', 'date': 'Thu, 20 Feb 2025 17:38:33 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'



======
Totals
======
Ran: 203 tests in 196.1381 sec.
 - Passed: 189
 - Skipped: 10
 - Expected Fail: 0
 - Unexpected Success: 0
 - Failed: 4
Sum of execute time for each test: 250.6245 sec.

==============
Worker Balance
==============
 - Worker 0 (47 tests) => 0:02:59.838559
 - Worker 1 (57 tests) => 0:03:13.406350
 - Worker 2 (99 tests) => 0:02:56.245418
